src/integrationTest/java/uk/gov/hmcts/reform/pcqbackend/controllers/GetPcqRecordWithoutCaseTest.java

```diff
@@ -0,0 +1,241 @@
+                DAYS_LIMIT + 1)));
+            pcqBackEndClient.createPcqAnswer(answerRequest);
+
+            //Now call the actual method.
+            Map<String, Object> responseMap = pcqBackEndClient.getPcqRecordWithoutCase();
+
+            //Test the assertions
+            assertTestForSuccess(responseMap, 0, answerRequest);
+            checkLogsForKeywords();
+
+
+        } catch (Exception e) {
+            log.error(EXCEPTION_MSG, e);
+        }
+
+    }
+
+    @Test
+    public void getPcqRecordWithoutCaseCompletedDateOnBoundary() {
+        try {
+
+            //Create the Test Data in the database.
+            String jsonStringRequest = jsonStringFromFile(JSON_FILE);
+            PcqAnswerRequest answerRequest = jsonObjectFromString(jsonStringRequest);
+            //Update the completed date to be in the past.
+            answerRequest.setCompletedDate(ConversionUtil.convertTimeStampToString(ConversionUtil.getDateTimeInPast(
+                DAYS_LIMIT)));
+            pcqBackEndClient.createPcqAnswer(answerRequest);
+
+            //Now call the actual method.
+            Map<String, Object> responseMap = pcqBackEndClient.getPcqRecordWithoutCase();
+
+            //Test the assertions
+            assertTestForSuccess(responseMap, 0, answerRequest);
+            checkLogsForKeywords();
+
+
+        } catch (Exception e) {
+            log.error(EXCEPTION_MSG, e);
+        }
+
+    }
+
+    @Test
+    public void getPcqRecordWithoutCaseCompletedDatePreBoundary() {
+        try {
+
+            //Create the Test Data in the database.
+            String jsonStringRequest = jsonStringFromFile(JSON_FILE);
+            PcqAnswerRequest answerRequest = jsonObjectFromString(jsonStringRequest);
+            //Update the completed date to be in the past.
+            answerRequest.setCompletedDate(ConversionUtil.convertTimeStampToString(ConversionUtil.getDateTimeInPast(
+                DAYS_LIMIT - 1)));
+            pcqBackEndClient.createPcqAnswer(answerRequest);
+
+            //Now call the actual method.
+            Map<String, Object> responseMap = pcqBackEndClient.getPcqRecordWithoutCase();
+
+            //Test the assertions
+            assertTestForSuccess(responseMap, 1, answerRequest);
+            checkLogsForKeywords();
+
+
+        } catch (Exception e) {
+            log.error(EXCEPTION_MSG, e);
+        }
+
+    }
+
+    @Test
+    public void getPcqRecordWithoutCaseMultipleIds() {
+        try {
+
+            //Create the Test Data 3 times in the database.
+            String jsonStringRequest = jsonStringFromFile(JSON_FILE);
+            PcqAnswerRequest answerRequest = jsonObjectFromString(jsonStringRequest);
+            answerRequest.setCompletedDate(updateCompletedDate(answerRequest.getCompletedDate()));
+            pcqBackEndClient.createPcqAnswer(answerRequest);
+
+            PcqAnswerRequest answerRequest2 = cloneAnswerObject(answerRequest,"INTEG-TEST-11", "PROBATE1",
```

Comment:
petitioner (divorce) and applicant (probate) wrong way round?

Reply:
Done